db/structure.sql

```diff
@@ -103,6 +103,43 @@ CREATE TABLE public.ar_internal_metadata (
 );
 
 
+--
```

Comment:
These top additions are just an automated rearrangement, not sure why they moved.